resources/views/robots-txt.phtml

```diff
@@ -27,7 +27,27 @@ Disallow: <?= $base_path ?>/admin
 Disallow: <?= $base_path ?>/manager
 Disallow: <?= $base_path ?>/editor
 Disallow: <?= $base_path ?>/account
-Crawl-delay: 10
+Crawl-delay: 1
+
+Disallow: <?= $base_path ?>/login   
+Disallow: <?= $base_path ?>/module 
```

Comment:
This will block access to the sitemap module.  We want robots to view it.

Reply:
this works for (only?) pretty urls since 3 month